components/raftstore/src/store/entry_storage.rs

```diff
@@ -147,6 +149,25 @@ impl EntryCache {
         }
     }
 
+    /// Push entries to the left of the cache.
+    ///
+    /// When cache is not empty, the index of the last entry in entries
+    /// should be equal to `cache first index - 1`. When cache is
+    /// empty, it should be equal to the store's last index. Otherwise,
+    /// append new entries may fail due to unexpected hole.
+    fn push_front(&mut self, entries: &[Entry]) {
```

Comment:
how about calling it `prepend`

Reply:
+1
push_front seems more used on single entry's push.

Reply:
fixed